api/src/main/java/org/pmiops/workbench/google/DirectoryServiceImpl.java

```diff
@@ -126,10 +123,12 @@ public User createUser(String givenName, String familyName, String username, Str
       .setEmails(new UserEmail().setType("custom").setAddress(contactEmail).setCustomType("contact"))
       .setChangePasswordAtNextLogin(true);
     retryHandler.run((context) -> getGoogleDirectoryService().users().insert(user).execute());
-    //This is a temporary solution until Mandrill is implemented.
-    //This way if one merge happens and the other is delayed for any reason,
-    //we won't break in a major way.
-    sendPasswordEmail(contactEmail, password, user);
+    try {
+      mailServiceProvider.get().sendWelcomeEmail(contactEmail, password, user);
+    } catch (MessagingException e) {
+      log.log(Level.SEVERE, "Welcome Email not sent: " + e.toString());
```

Comment:
I'm wondering if we need to log a bit more info here. In this case, we've swallowed the password so it will never be seen again. We also don't know who the user is, unless it's part of the message exception. In the case where we've lost the PW, I think that's a manageable situation because an admin can reset that in the gsuite interface. We still need to know the user so we can then follow up on that from a bug reporting standpoint.

Let's add some more info about the user that tried to register here. At a minimum, we need the contact email and the gsuite username somewhere in the message. When the exception eventually makes its way into a jira ticket, then we should have everything we need to follow up.